docs/topics/native/native-libraries.md

```diff
@@ -1,194 +1,193 @@
+| `info`                        | General information about the library.                                                                                                                                                                                                                                                                                                                                |
+| `dump-abi`                    | Dump the ABI snapshot of the library. Each line in the snapshot corresponds to one declaration. In case an ABI-incompatible change happens to a declaration, it'll be visible in the corresponding line of the snapshot.                                                                                                                                              |
+| `dump-ir`                     | Dump the intermediate representation (IR) of library declarations to the output. Use only for debugging.                                                                                                                                                                                                                                                              |
+| `dump-ir-signatures`          | Dump IR signatures of all non-private library declarations and all non-private declarations consumed by this library (as two separate lists). This command relies purely on the data in IR.                                                                                                                                                                           |
+| `dump-ir-inlinable-functions` | Dump the intermediate representation (IR) of inlinable functions in the library to the output. Use only for debugging.                                                                                                                                                                                                                                                |
+| `dump-metadata`               | Dump the metadata of all library declarations to the output. Use only for debugging.                                                                                                                                                                                                                                                                                  |
+| `dump-metadata-signatures`    | Dump IR signatures of all non-private library declarations based on the library metadata. Comparing it to the `dump-ir-signatures` command, which renders signatures based on IR, in most cases the outputs are the same, but if IR-transforming compiler plugins (such as Compose) were used during compilation, patched declarations may have different signatures. |
 
-```kotlin
-package kotlinizer
-val String.kotlinized
-    get() = "Kotlin $this"
-```
+All the above commands accept an additional `-repository <directory>` argument for specifying a repository different
```

Comment:
`-repository` has been removed some time ago. You can mention `-signature-version` and `-print-signatures` instead.